.github/workflows/ios-staging.yml

```diff
@@ -0,0 +1,106 @@
+name: Release iOS Staging
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - '*'
```

Comment:
before we merge this we probably only want this on `workflow_dispatch`, don't need a signet release for every pr and commit